ui/src/app/workflows/components/workflows-toolbar/workflows-toolbar.tsx

```diff
@@ -10,66 +11,71 @@ require('./workflows-toolbar.scss');
 
 interface WorkflowsToolbarProps {
     selectedWorkflows: Map<string, Workflow>;
-    loadWorkflows: () => void;
-    isDisabled: Actions.OperationDisabled;
+    actionsIsDisabled: Actions.OperationDisabled;
```

Comment:
```suggestion
    disabledActions: Actions.OperationDisabled;
```

that probably makes a bit more sense since it's a mapping